.github/workflows/docker.yaml

```
@@ -49,43 +49,57 @@ jobs:
       with:
         authToken: '${{ secrets.CACHIX_CARDANO_SCALING_AUTH_TOKEN }}'
 
-    - name: 🔨 Build image using nix
+    - name: 🔨 🏷️ Build and tag images
       run: |
+
         IMAGE_NAME=ghcr.io/${{github.repository_owner}}/${{matrix.target}}
         echo "IMAGE_NAME=${IMAGE_NAME}" >> $GITHUB_ENV
         nix build .#docker-${{ matrix.target }}
         docker load < result
 
-        # Determine whether we are building a tag and if yes, set a VERSION_NAME
+        # Compute tags. We want the following behaviour:
+        #
+        # 1. Building on tag:      image tagged `tag`
+        # 2. Building on release:  image tagged `release`
```

i think here is a typo, isn't it suppose to be label as `latest` ?

```suggestion
        # 2. Building on release:  image tagged `latest`
```